frontend/packages/pipelines-plugin/src/utils/hooks.ts

```diff
@@ -1,12 +1,21 @@
 import { BadgeType, getBadgeFromType } from '@console/shared';
 import {
   isGAVersionInstalled,
+  isTriggersGAVersionInstalled,
   usePipelineOperatorVersion,
 } from '../components/pipelines/utils/pipeline-operator';
 
+const getOSPTechPreviewBadge = (installedGAVersion: boolean) =>
+  installedGAVersion ? null : getBadgeFromType(BadgeType.TECH);
+
 export const usePipelineTechPreviewBadge = (namespace: string) => {
   const operator = usePipelineOperatorVersion(namespace);
   if (!operator) return null;
-  const installedGA = isGAVersionInstalled(operator);
-  return installedGA ? null : getBadgeFromType(BadgeType.TECH);
+  return getOSPTechPreviewBadge(isGAVersionInstalled(operator));
+};
+
+export const useTriggersTechPreviewBadge = (namespace: string) => {
```

Comment:
Should we not use a generic `getBadge` function and return badge using `getBadgeFromType`?? WDYT??

Reply:
The utility function to return the badges `getOSPTechPreviewBadge` internally uses getBadgeFromType function.